### ISSUE TITLE
training GaitGL on OUMVLP with default config too slow  to tolerate.....

### ISSUE THREAD

**Issue by silverlining21** [ShiqiYu/OpenGait#7]
I try GaitGL on `OUMVLP` dataset, 6000 id for training and rest for teting, using deafault config with Tesla v100*8, it's cost 165 seconds for each 100 iters. if I want 10 epochs, `time = 10(epoch) x 130000(seqs) / 8(bs) / 100 * 165 = 3.1 days`.

I guess computing  triplet loss cost might cost lot of time, I'm tring to modify the train procedure to, triplet loss on take effect at latter epochs, while early epochs only compute id loss, namely softmaxloss.

Is there any other methods to speed up?

thxs
